docs/api_refs/package.json

```diff
@@ -5,10 +5,10 @@
   "scripts": {
     "dev": "next dev -p 3001",
     "typedoc": "npx typedoc --options typedoc.json",
-    "build:scripts": "yarn typedoc && node ./scripts/update-typedoc-css.js",
+    "build:scripts": "node ./scripts/create-entrypoints.js && yarn typedoc && node ./scripts/update-typedoc-css.js",
     "build": "yarn clean && yarn run build:deps && yarn build:scripts && next build",
     "build:vercel": "yarn clean && yarn run build:deps --force && yarn build:scripts && next build",
-    "build:deps": "yarn run turbo:command build --filter=@langchain/core --filter=@langchain/anthropic --filter=@langchain/openai --filter=@langchain/community --filter=langchain --concurrency=1",
+    "build:deps": "yarn run turbo:command build --filter=@langchain/core --filter=@langchain/anthropic --filter=@langchain/openai --filter=@langchain/community --filter=@langchain/mistralai --filter=@langchain/google-genai --filter=langchain --concurrency=1",
```

Comment:
Will we need to continue to add packages here?

Reply:
Yes, it might make more sense to turn this into a script which builds all packages inside `./libs` (excluding the template repo), langchain, and langchain core.

Reply:
Or do we need the filter at all if we want to build everything?

Reply:
We don't want to waste time building things like examples, core_docs, and other pkgs which don't matter to api refs